### PR TITLE
[bugfix] give /tmp a bit more room for when the CI VM gets busy

### PR DESCRIPTION
**Pull request by s-amann**
### Which issue this PR addresses:

[ADO WI](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/13983121)

### What this PR does / why we need it:

/tmp fills up on some VMs during builds/tests

### Test plan for issue:

n/a

### Is there any documentation that needs to be updated for this PR?

n/a

### CODE REVIEW

File: pkg/deploy/generator/resources_dev.go [hunk 1]
```diff
@@ -306,7 +306,10 @@ for attempt in {1..5}; do
   if [[ ${attempt} -lt 5 ]]; then sleep 10; else exit 1; fi
 done
 
-lvextend -l +50%FREE /dev/rootvg/varlv
+lvextend -l +25%FREE /dev/rootvg/tmplv
+xfs_growfs /tmp
+
+lvextend -l +25%FREE /dev/rootvg/varlv
```
Comment (bennerv) [inline pkg/deploy/generator/resources_dev.go:312]: I'm not sure if this is doing what you expect.  

Let's assume there's 1GB of free space:
1. `lvextend -l +25%FREE /dev/rootvg/tmplv` -> adds 0.25GB of space to the /tmp filesystem
2. `lvextend -l +25%FREE /dev/rootvg/varlv` -> adds 25% of remaining space (0.75GB * 25% = .1875GB) to /var
3. `lvextend -l +100%FREE /dev/rootvg/homelv` -> adds all remaining free space to the /home directory `1 - 0.25 - .1875)

I think this is how it works.  If this is the case, you're going to want to do this in reverse order with the following
1. `lvextend -l +50%FREE /dev/rootvg/homelv`
2. `lvextend -l +50%FREE /dev/rootvg/tmplv`
3. `lvextend -l +100%FREE /dev/rootvg/varlv`

**Review by bennerv — COMMENTED** [review]
Question on this.  not sure how `lvextend` works and I think the logic here might be wrong.

File: pkg/deploy/generator/resources_dev.go (hunk 1 above)
Comment (s-amann) [reply pkg/deploy/generator/resources_dev.go:312]: Ah, right. Thank you, I'll update.

Comment (bennerv) [reply pkg/deploy/generator/resources_dev.go:312]: Yeah from the [man page](https://linux.die.net/man/8/lvextend)

The number can also be expressed as a percentage of the remaining free space in the Volume Group with the suffix %FREE

**Review by cadenmarchese — APPROVED** [review]
I like that we're using percent free so that we're not tied to a specific VM storage size. Thanks!

**Review by bryanro92 — APPROVED** [review]

**Review by ellis-johnson — APPROVED** [review]
lgtm!

**Review by bennerv — APPROVED** [review]

**Pull request merged**